### ISSUE TITLE
Document or add support for elementVariable and collection in Flowable 6 Multi-Instance

### ISSUE THREAD

**Issue by Adrian-Bielefeldt** [bpmn-io/bpmn-js-properties-panel#760]
__Is your feature request related to a problem? Please describe__

I need to be able to set `multiInstanceLoopCharacteristics` with `elementVariable` and `collection` attributes on a service task. I can set the property via the change type context menu, which also creates a Multi-Instance section in the properties panel. However, this only lists two attributes, `Loop cardinality` and `Completion condition`. The attributes I need are not present.

__Describe the solution you'd like__

I would like some option to add / activate these two attributes, `elementVariable` and `collection`. Specifically, I would like to know how the image in this [issue](https://github.com/bpmn-io/bpmn-js/issues/1581) came to be. Is this a custom implementation, or can I somehow configure my properties panel to display and edit these attributes?

I also saw another [issue](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/105) with a screenshot showing the attributes I need. This seems to be an older version of the properties panel, though.

__Describe alternatives you've considered__

I considered adding a custom moddle and properties provider to edit these on my own, but since Multi-Instance is supported out of the box, it seems less than ideal to ignore that and go my own way.

__Additional context__

Basically it should look like this ([source](https://github.com/bpmn-io/bpmn-js/issues/1581))
![150531680-3f51471b-df5f-4b08-b6c1-d49b2e6a2209](https://user-images.githubusercontent.com/21172799/190158622-5567b512-4127-4e5a-b995-f611f6f35385.gif)

**Comment by pinussilvestrus** [comment]
Which properties provider are you using? `bpmn`, `camunda-platform` or `zeebe`?

**Comment by Adrian-Bielefeldt** [comment]
I'm using `BpmnPropertiesProviderModule`.

**Comment by pinussilvestrus** [comment]
`elementVariable` and `collection` are not part of the BPMN 2.0 standard. Therefore the `BpmnPropertiesProviderModule` does not include them, these are specific to the [Camunda Platform 7 moddle extensions](https://github.com/camunda/camunda-bpmn-moddle/blob/master/resources/camunda.json#L796). 

If you can't use the `CamundaPlatformPropertiesProvder` you need to create a [custom properties provider and the necessary moddle extension](https://github.com/bpmn-io/bpmn-js-examples/tree/master/properties-panel-extension).

I'm closing this one as this is expected behavior. Feel free to re-open in case you have further information (or head over to the forum in case of any question).
